src/jimgw/single_event/runManager.py

```diff
@@ -122,6 +123,14 @@ def save(self, path: str):
     def load_from_path(self, path: str) -> SingleEventRun:
         with open(path, "r") as f:
             data = yaml.safe_load(f)
+        if (
+            "jim_parameters" in data
+            and "local_sampler_arg" in data["jim_parameters"]
+            and "step_size" in data["jim_parameters"]["local_sampler_arg"]
+        ):
+            data["jim_parameters"]["local_sampler_arg"]["step_size"] = jnp.array(
+                data["jim_parameters"]["local_sampler_arg"]["step_size"]
+            )
```

Comment:
May I know why do we need this? Btw, you can do this with a `try` block, it would probably look cleaner.

Reply:
Usually the local_sampler_arg requires input something like this:
```
mass_matrix = jnp.eye(11)
mass_matrix = mass_matrix.at[1, 1].set(1e-3)
mass_matrix = mass_matrix.at[5, 5].set(1e-3)
mass_matrix = mass_matrix * 3e-3
local_sampler_arg = {"step_size": mass_matrix}
```
If this line is not included, `local_sampler_arg` will be loaded as `list` instead of `jnp.array`. This causes bugs in compiling the sampler.

Reply:
What do you mean by the `try` block implementation? I think I could change it to an `if` statement though, to check if there is `local_sampler_arg`.

Reply:
Ohh, nice catch. I would personally still do it with `try` and probably an exception with key error. Other than that, this looks good.

Reply:
Something like
```
try:
    data[...] = jnp.array(data[...])
except KeywordForKeyError: # replace with the correct error name
    pass
```

Reply:
I mean you can do it without the `if` statement. As long as, that key is not found, it will execute the `except` block. For the exception, instead of all error, you should only except the key error.